gm4_metallurgy/data/gm4_metallurgy/functions/tick.mcfunction

```diff
@@ -6,8 +6,7 @@ execute if score found_item_on_anvil gm4_ml_data matches 1 as @e[type=item,tag=g
 execute if entity @e[tag=gm4_in_spiraculum] as @a[tag=gm4_has_spiraculum] at @s run function gm4_spiraculum_shamir:spiral
 
 # arborenda
-execute if score arborenda_active gm4_ml_data matches 1 as @a[tag=gm4_has_arborenda] at @s run function gm4_arborenda_shamir:main
-execute as @e[type=area_effect_cloud,tag=gm4_arborenda_trunk] at @s run function gm4_arborenda_shamir:destroy_trunk
+function gm4_arborenda_shamir:main
```

Comment:
It concerns me slightly that this will run every tick with no limits, but I cannot think of any good ways to do so. I guess this comment is just a "double checking that this can't be improved" reminder

Reply:
All it does is check scores so it should be very efficient, even though it has a lot of selectors. If we notice it's a problem we can upgrade it to a tick advancement checking the scores.